resources/testdata/glyphs2/Fea_Order.glyphs

```diff
@@ -0,0 +1,69 @@
+{
+.appVersion = "3148";
+classes = (
```

Comment:
I think you have `classes` entry twice (there's another one probably overwriting this one further below), keys in plist dicts are unique

Reply:
Oops, indeed.